.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Grain NEAR SDK CI Workflow
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Test
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ["16"]
+        grain-version: ["v0.5.11"]
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3.1.1
```

Comment:
Can probably use a newer version of this unless you were trying to align things (?).. Might want to use the versions referenced by binaryen.ml or something?